tcms/testplans/static/testplans/js/get.js

```diff
@@ -1,6 +1,7 @@
 var expandedTestCaseIds = [];
 
-const allTestCases = {};
+const allTestCases = {},
+      permissions = {};
```

Comment:
The `permissions` variable seems to be unused, no ?

Reply:
I added it, because I needed it on global level, but it looks like I can pass it as field parameter.